data/json/recipes/recipe_deconstruction.json

```diff
@@ -506,6 +506,60 @@
     ]
   },
   {
+    "result": "makeshift_oscilloscope",
```

Comment:
If you felt like doing a github wiki page or `doc/` entry I suspect it would be greatly appreciated